underfs/swift/src/main/java/alluxio/underfs/swift/SwiftInputStream.java

```diff
@@ -102,23 +115,39 @@ public long skip(long n) throws IOException {
    * Opens a new stream at mPos if the wrapped stream mIn is null.
    */
   private void openStream() {
+    LOG.debug("Swift InputStream {}: open stream at pos {}", this.hashCode(), mPos);
+
     if (mStream != null) { // stream is already open
       return;
     }
     StoredObject storedObject = mAccount.getContainer(mContainerName).getObject(mObjectPath);
     DownloadInstructions downloadInstructions  = new DownloadInstructions();
-    downloadInstructions.setRange(new ExcludeStartRange(mPos));
+    final long blockSize = getBlockSize();
+    final long endPos = mPos + blockSize - (mPos % blockSize);
+    downloadInstructions.setRange(new SwiftRange(mPos, endPos));
     mStream = storedObject.downloadObjectAsInputStream(downloadInstructions);
+    LOG.debug("Swift InputStream {}: stream open till end pos {}", this.hashCode(), endPos);
   }
 
   /**
    * Closes the current stream.
    */
   private void closeStream() throws IOException {
+    LOG.debug("Swift InputStream {}: closing stream at pos {}", this.hashCode(), mPos);
     if (mStream == null) {
       return;
     }
     mStream.close();
     mStream = null;
+    LOG.debug("Swift InputStream {}: stream closed", this.hashCode());
```

Comment:
Let's remove the debugging statements? Or maybe just keep a few you think are especially valuable?